openshift_scalability/ci/contents/nodeVertical-500-pods-ci.yaml

```diff
@@ -0,0 +1,27 @@
+projects:
+  - num: 1
+    basename: clusterproject
+    ifexists: delete
```

Comment:
@mffiedler I can update the nodeVertical.yaml file to add the ifexists: delete line.  We have been keeping copies of yaml config files in the ci/content subdir so scripts and config files are both contained in the ci dir.  Do we only want to keep scripts in the ci dir ?